src/blockchain/tx-history-service.js

```diff
@@ -0,0 +1,299 @@
+'use strict';
+
+import request from 'request';
+import AsyncTaskQueue from '../utils/async-task-queue';
+import BigNumber from 'bignumber.js';
+
+import { Logger } from '../logger';
+import config from '../config';
+import Wallet from '../wallet/wallet';
+import WalletSetting from '../wallet/wallet-setting';
+import TxHistory from './tx-history';
+
+const log = new Logger('tx-history-service');
+
+export const REQUEST_INTERVAL_DELAY = 600; // millis
+export const ETH_BALANCE_DIVIDER = new BigNumber(10 ** 18);
+export const ENDPOINT_CONFIG = {
+	1: { url: 'https://api.etherscan.io/api' },
+	3: { url: 'http://api-ropsten.etherscan.io/api' }
+};
+export const API_ENDPOINT = ENDPOINT_CONFIG[config.chainId].url;
+
+export let OFFSET = 1000;
+
+export const TX_LIST_ACTION = `?module=account&action=txlist&sort=asc&offset=${OFFSET}`;
```

Comment:
@sk91 can you update the entire file with current dev version please? 
https://github.com/SelfKeyFoundation/Identity-Wallet/blob/dev/src/main/controllers/tx-history-service.js